src/core/file_format/trp.cpp

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "trp.h"
+
+TRP::TRP() = default;
+TRP::~TRP() = default;
+
+void TRP::GetNPcommID(std::filesystem::path trophyPath, int index) {
+    std::filesystem::path trpPath = trophyPath / "sce_sys/npbind.dat";
+    Common::FS::IOFile npbindFile(trpPath, Common::FS::FileAccessMode::Read);
+    if (!npbindFile.IsOpen()) {
+        return;
+    }
+    npbindFile.Seek(0x84 + (index * 0x180));
+    npbindFile.Read(NPcommID);
+    NPcommID.resize(16);
```

Comment:
This is one use of ReadRaw xD
```suggestion
    npbindFile.ReadRaw<u8>(np_comm_id.data(), 12);
```